### PR TITLE
API for merging fixture rows

### PR DESCRIPTION
**Pull request by vkorobkov**
This is the first step and a dependency for merging _fixtures_, which would allow multiple fixture sources living together, like DSL+DSL, YML+DSL and so on

### CODE REVIEW

File: src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java [hunk 1]
```diff
@@ -15,4 +19,11 @@ public Fixture(String name, Collection<FixtureRow> rows) {
         this.name = name;
         this.rows = Collections.unmodifiableCollection(rows);
     }
+
+    public Fixture mergeRows(Collection<FixtureRow> rows) {
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java]: Can we rename `rows` with something like `newRows`? `mergingRows`?

**Review by rodionovsasha — APPROVED** [review]

File: src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java (hunk 1 above)
Comment (vkorobkov) [reply src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java]: Why? Because it overlaps with `this.rows`?

Comment (rodionovsasha) [reply src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java]: yes, looks a bit unclear `this.rows.stream(), rows.stream()`

Comment (vkorobkov) [reply src/main/java/com/github/vkorobkov/jfixtures/loader/Fixture.java]: Fixed

**Pull request merged**